presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -395,9 +404,17 @@ public void testDropColumn()
         assertUpdate("CREATE TABLE " + tableName + " AS SELECT 123 x, 111 a", 1);
 
         assertUpdate("ALTER TABLE " + tableName + " DROP COLUMN x");
+        assertUpdate("ALTER TABLE " + tableName + " DROP COLUMN IF EXISTS notExistColumn");
```

Comment:
Add test where actually column is dropped when `IF EXISTS` is used.

Reply:
fixed thanks!

Reply:
updated for other test case as well.